doc/rtd/explanation/format.rst

```diff
@@ -159,14 +159,60 @@ a MIME archive.
 ``cloud-boothook``
 ==================
 
-This content is `boothook` data. It is stored in a file under
-:file:`/var/lib/cloud` and executed immediately. This is the earliest `hook`
-available. Note, that there is no mechanism provided for running only once. The
+This content is `boothook` data.
+This is a bash script that runs during every boot.
+This is the earliest `hook` to run, it runs during the `Network boot stage`.
```

Comment:
What is a hook? What other hooks exist? Can we link to a definition with a list of hooks please?

Using undefined terms is confusing.